src/main.rs

```diff
@@ -468,6 +481,43 @@ impl From<NonNegativeI64> for u64 {
     }
 }
 
+fn print_color_table() -> io::Result<()> {
+    let stdout = io::stdout();
+    let mut stdout_lock = BufWriter::new(stdout.lock());
```

Comment:
I know we use this pattern already, but this could be
```rs
let mut out = BufWriter::new(io::stdout().lock());
```
in recent versions of Rust.